capmvm/kernel/Makefile

```diff
@@ -0,0 +1,21 @@
+DOCKER := docker
+
+REGISTRY?=weaveworks
+IMAGE_NAME?=$(REGISTRY)/capmvm-kernel
+BUILDER_IMAGE_NAME?=$(REGISTRY)/capmvm-kernel-builder
+KERNEL_VERSIONS ?= 4.19.215 5.10.77 # If you update this list, please remember to update the matrix in the kernel-images.yml github action!
+
+all: build
+
+kernel-builder:
+	$(DOCKER) build -t $(BUILDER_IMAGE_NAME):dev \
+		-f builder/Dockerfile .
+
+build: $(addprefix build-,$(KERNEL_VERSIONS))
+build-%: kernel-builder
+	$(DOCKER) build -t $(IMAGE_NAME):$* \
+		--build-arg KERNEL_VERSION=$* .
```

Comment:
See comment above about passing in the registry for the kernel builder base image